### PR TITLE
chore(curriculum): add url shortener lab

### LINKED ISSUE #562: chore: add URL Shortener Microservice project to frontend cert
The frontend cert will have a brief section to talk about and practice working with node and express.

Instead of a traditional lab here, we will use the existing URL Shortener Microservice.

https://www.freecodecamp.org/learn/back-end-development-and-apis/back-end-development-and-apis-projects/url-shortener-microservice

### PR DESCRIPTION
**Pull request by majestic-owl448**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes https://github.com/freeCodeCamp/CurriculumExpansion/issues/562

ALERT for merge conflicts for the duplicated IDs

### CODE REVIEW

**Review by jdwilkin4 — APPROVED** [review]

File: curriculum/challenges/english/25-front-end-development/lab-url-shortener-microservice/bd7158d8c443edefaeb5bd0e.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+---
+id: bd7158d8c443edefaeb5bd0e
+title: URL Shortener Microservice
+challengeType: 4
```
Comment (moT01) [inline curriculum/challenges/english/25-front-end-development/lab-url-shortener-microservice/bd7158d8c443edefaeb5bd0e.md:4]: I noticed the forumTopicId was removed here. Do we not want that anymore? I feel like we should keep it.

File: curriculum/challenges/_meta/lab-url-shortener-microservice/meta.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "name": "Build an URL Shortener Microservice",
+    "isUpcomingChange": true,
+    "dashedName": "lab-url-shortener-microservice",
+    "order": 375,
+    "superBlock": "front-end-development",
+    "challengeOrder": [{ "id": "bd7158d8c443edefaeb5bd0e", "title": "Build an URL Shortener Microservice" }],
```
Comment (moT01) [inline curriculum/challenges/_meta/lab-url-shortener-microservice/meta.json:7]:
```suggestion
    "challengeOrder": [{ "id": "bd7158d8c443edefaeb5bd0e", "title": "Build a URL Shortener Microservice" }],
```

File: curriculum/challenges/_meta/lab-url-shortener-microservice/meta.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+    "name": "Build an URL Shortener Microservice",
```
Comment (moT01) [inline curriculum/challenges/_meta/lab-url-shortener-microservice/meta.json:2]:
```suggestion
    "name": "Build a URL Shortener Microservice",
```

File: client/i18n/locales/english/intro.json [hunk 4]
```diff
@@ -2229,7 +2229,10 @@
       "sykw": { "title": "372", "intro": [] },
       "zzhp": { "title": "373", "intro": [] },
       "nmve": { "title": "374", "intro": [] },
-      "agky": { "title": "375", "intro": [] },
+      "lab-url-shortener-microservice": {
+        "title": "Build an URL Shortener Microservice",
```
Comment (moT01) [inline client/i18n/locales/english/intro.json:2233]:
```suggestion
        "title": "Build a URL Shortener Microservice",
```

File: client/i18n/locales/english/intro.json [hunk 5]
```diff
@@ -2229,7 +2229,10 @@
...
+        "intro": ["In this lab, you will build an URL Shortener Microservice"]
```
Comment (moT01) [inline client/i18n/locales/english/intro.json:2234]:
```suggestion
        "intro": ["In this lab, you will build a URL Shortener Microservice"]
```

File: client/src/pages/learn/front-end-development/lab-url-shortener-microservice/index.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
+---
+title: Introduction to the Build an URL Shortener Microservice
```
Comment (moT01) [inline client/src/pages/learn/front-end-development/lab-url-shortener-microservice/index.md:2]:
```suggestion
title: Introduction to the Build a URL Shortener Microservice
```

File: client/src/pages/learn/front-end-development/lab-url-shortener-microservice/index.md [hunk 7]
```diff
@@ -0,0 +1,9 @@
...
+block: lab-url-shortener-microservice
+superBlock: front-end-development
+---
+
+## Introduction to the Build an URL Shortener Microservice
+
+In this lab you will build an URL Shortener Microservice
```
Comment (moT01) [inline client/src/pages/learn/front-end-development/lab-url-shortener-microservice/index.md:9]:
```suggestion
In this lab you will build a URL Shortener Microservice
```

File: client/src/pages/learn/front-end-development/lab-url-shortener-microservice/index.md [hunk 8]
```diff
@@ -0,0 +1,9 @@
+---
+title: Introduction to the Build an URL Shortener Microservice
+block: lab-url-shortener-microservice
+superBlock: front-end-development
+---
+
+## Introduction to the Build an URL Shortener Microservice
```
Comment (moT01) [inline client/src/pages/learn/front-end-development/lab-url-shortener-microservice/index.md:7]:
```suggestion
## Introduction to the Build a URL Shortener Microservice
```

**Review by moT01 — CHANGES_REQUESTED** [review]
Found one typo across many of the files. Also left one concern.

Other than that, LGTM 👍

File: curriculum/challenges/_meta/lab-url-shortener-microservice/meta.json (hunk 3 above)
Comment (ilenia-magoni) [reply curriculum/challenges/_meta/lab-url-shortener-microservice/meta.json:2]: English is weird

**Review by moT01 — APPROVED** [review]

**Pull request merged**